backend/tests/test_integration.py

```diff
@@ -2,7 +2,7 @@
 import requests
 
 class TestQueueIntegration:
-    host = 'http://backend:5000'
+    host = 'http://localhost:5000'
```

Comment:
this will break docker-compose test runs

Reply:
Fixed using a default value if no environment variable has is specified